reports/summary-2020.md

```diff
@@ -0,0 +1,814 @@
+
+### #shoutouts
+
+A meta accolade.
+
+The #shoutouts channel on the [Kubernetes slack](slack.k8s.io) and highlights
+on [@k8scontributors] in the last year has kept us going. Thank a member of the
+community here and read the past achievements of many. 
+
+[@k8scontributors]: https://twitter.com/K8sContributors
+
+### Remembering Dan Kohn
+
+In [November of 2020, Dan Kohn], the former director of the CNCF sadly passed
+away from complications with Colon Cancer. Dan was instrumental in shaping both
+the Kubernetes project, the CNCF, and the Cloud Native community as a whole. He
+understood that a foundation built on a vibrant and diverse community was a
+requirement to be successful, and the project would not be what it is today
+without him. 
+
+[November of 2020, Dan Kohn]: https://kubernetes.io/blog/2020/11/02/remembering-dan-kohn/
+
+
+## Themes
+
+The following themes emerged from multiple community groups reporting in with 
+similar experiences - whether positive or challenging - and areas of research to
+explore more in the future. 
+
+### Project communication strategy 
+  
+During major growth phases, the Steering Committee needed the various [community
+groups to report in and out] regularly with the group's members, the
+project at large, and at KubeCons. This added up to many duplicate meetings and 
+many update slides. The
+COVID-19 pandemic and an increase in contributors outside of the
+North America Pacific Timezone made regular meetings difficult. Chairs and other
+project leads asked for a more streamline and consistent
+reporting and feedback mechanism. 
+    
+#### What we've done
+
+- Changed [Community Meeting] cadence from weekly to monthly, and changed 
+meeting style from a "read out" of updates to more discussion oriented.
+- Encouraged groups to use asychronous methods for delivering updates and 
+gathering feedback. For example: Slack "standup" theads which feed into larger 
+scale reporting to a group on a mailing list.
+- Created an internal marketing group under SIG Contributor Experience to help
+  facilitate community communication.
+
+
+[Community Meeting]: https://k8s.dev/events/community-meeting/
+[community groups to report in and out]: https://git.k8s.io/community/committee-steering/governance/sig-governance.md
+[Governance requirements]: https://git.k8s.io/community/committee-steering/governance/sig-governance.md#chair
+    
+#### Areas of research 
+
+- Review SIG charters to re-focus goals of each group at the current state of
+  maturity
+- Expand on async meeting adoption and establish best practices 
+- Improving subproject communication and connective glue back to the sponsoring
+  SIG
+- Clarify conditions for archiving or putting a subproject in maintenance mode
+- Connecting with the end user community and understanding what they want to
+  hear from upstream
+  
+    
+### Refining the Contributor Lifecycle
```

Comment:
I don't like that this section doesn't follow the same pattern as the others under "Themes". 

The others follow a pattern:

[ Name ]
- Description
- What we've done
- Areas for research

This one has subsections which don't really describe "what we've done" in the same way? It feels like lots of text with no real point to it.

----

Having closely edited my way through this section now... this feels so different than the rest of the document that I wonder if it's its own top-level section, honestly, or even a sub-section under "Contributor Snapshot"?

IMHO: Either parallelize the language/structure of this section to be in line with the other themes as suggested above, OR move this to its own top-level section or a section under contributor snapshot and do a little less editing.

Reply:
I updated this one - take a look